### ISSUE TITLE
multi_column_addons.css

### ISSUE THREAD

**Issue by wincmd64** [MrOtherGuy/firefox-csshacks#399]
https://github.com/MrOtherGuy/firefox-csshacks/blob/master/content/multi_column_addons.css

does it work on ESR v115.14 ?

I'm tried to add `@import url(multi_column_addons.css);` to `userContent.css`
but nothing has changed.

**Comment by MrOtherGuy** [comment]
I don't know about ESR 115, but it works in Firefox 129 and considering that last update to that style is from 3 years ago so I'd say it very likely that it works in ESR 115 as well.

I think it's more likely that your import is not working - perhaps because `@import` statements must be placed before anything else in the file.

**Comment by wincmd64** [comment]
ok, thx, it's helped.
can you make all the cells the same height?

![add](https://github.com/user-attachments/assets/220f069c-eb2e-47f4-9fcd-2b378ede2634)

**Comment by MrOtherGuy** [comment]
Yes, you can, but what exactly do you want to happen here? Should the taller ones collapse or should the items that have less text expand more? And should *all* items be equal height or just the items on same row? What about items like that sponsorblock - should it's name be clipped since it is taller all by itself than the Selection Search card next to it?

**Comment by wincmd64** [comment]
To be honest, I don't know what's best.
But the way it is now - is no good.

**Comment by MrOtherGuy** [comment]
Well, you can add this if you want to try:
```css
addon-card:not([expanded]){ display: flex; }
.addon.card{ flex-grow: 1 }
```

I'm not convinced it's better than the current layout - but at least it isn't definitely worse, unlike the other alternatives I mentioned.
